search/controllers/api/__init__.py

```diff
@@ -304,18 +308,22 @@ def _get_classification(value: str, field: str, query_terms: List) \
 }
 
 
-def _parse_search_query(query: str) -> Dict[str, Any]:
-    # TODO: Add support for booleans.
+def _parse_search_query(query: str) -> Tuple[Dict[str, Any], Dict[str, Any]]:
+    """Parses a query into tuple of operators and parameters."""
     new_query_params = {}
+    new_query_operators: Dict[str, str] = defaultdict(default_factory=lambda: "AND")
     terms = query.split()
 
     expect_new = True
     """expect_new handles quotation state."""
+    next_operator = "AND"
+    """new_bool handles the operator state """
```

Comment:
Curious why these are docstrings? Would expect these to be inline comments

Reply:
now inline comments